fastlane/swift/SocketClient.swift

```diff
@@ -224,6 +248,7 @@ extension SocketClient: StreamDelegate {
         } else if aStream === self.outputStream {
             switch eventCode {
             case Stream.Event.openCompleted:
+                print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "Leave \(#function) \(#line)")
```